Meta/Lagom/Tools/CodeGenerators/GMLCompiler/main.cpp

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2023, kleines Filmröllchen <filmroellchen@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Forward.h>
+#include <AK/String.h>
+#include <AK/Try.h>
+#include <AK/Utf8View.h>
+#include <LibCore/Stream.h>
+#include <LibGUI/GML/Parser.h>
+#include <LibMain/Main.h>
+
+ErrorOr<int> serenity_main(Main::Arguments arguments)
+{
+    if (arguments.argc != 2) {
+        outln("usage: {} <GML file>", arguments.strings[0]);
+        return 1;
+    }
+
+    auto gml_file_name = arguments.strings[1];
```

Comment:
Let's not get into the habit of doing direct `arguments` access, use an ArgsParser for nice help & stuff.

Reply:
That's another FIXME for the other code generators

Reply:
https://github.com/SerenityOS/serenity/blob/4d68dcfca0487b7cc0c421932f4d4a1c8c44ee5b/Meta/Lagom/Tools/CodeGenerators/LibTimeZone/GenerateTimeZoneData.cpp#L815
https://github.com/SerenityOS/serenity/blob/4d68dcfca0487b7cc0c421932f4d4a1c8c44ee5b/Meta/Lagom/Tools/CodeGenerators/LibLocale/GeneratePluralRulesData.cpp#L669
https://github.com/SerenityOS/serenity/blob/4d68dcfca0487b7cc0c421932f4d4a1c8c44ee5b/Meta/Lagom/Tools/CodeGenerators/LibUnicode/GenerateUnicodeData.cpp#L1510
https://github.com/SerenityOS/serenity/blob/4d68dcfca0487b7cc0c421932f4d4a1c8c44ee5b/Meta/Lagom/Tools/CodeGenerators/LibWeb/BindingsGenerator/main.cpp#L32
...